terraform/environment/region/variables.tf

```diff
@@ -152,3 +152,8 @@ variable "fault_injection_experiments_enabled" {
   type        = bool
   description = "Enable fault injection"
 }
+
+variable "search_endpoint" {
+  type        = string
+  description = "URL of the OpenSearch Service endpoint to use"
```

Comment:
Because we know that there's no service in London, perhaps we can make this nullable for now? https://developer.hashicorp.com/terraform/language/values/variables#disallowing-null-input-values